src/test/java/uk/gov/pay/adminusers/infra/LedgerStub.java

```diff
@@ -34,4 +41,17 @@ public void returnLedgerTransaction(String externalId, LedgerTransaction ledgerT
                         .willReturn(response)
         );
     }
+
+    public void returnLedgerTransactionsForSearch(String gatewayAccountId, LedgerSearchTransactionsResponse ledgerResponse) throws JsonProcessingException {
+        ResponseDefinitionBuilder response = aResponse()
+                .withHeader(CONTENT_TYPE, APPLICATION_JSON)
+                .withStatus(200)
+                .withBody(objectMapper.writeValueAsString(ledgerResponse));
+        wireMockExtension.stubFor(
```

Comment:
Is the `wireMockExtension.stubFor` used here instead of `wireMockServer.stubFor` because `wireMockServer` doesn't work with Junit 5 tests?

Reply:
Yeah using Extension because of Junit5.  We are using `@Rule` elsewhere for wireMockServer which doesn't work with Junit5.